log/config.go

```diff
@@ -105,27 +105,45 @@ func init() {
 
 // prepZap is a utility function used by the Configure function.
 func prepZap(options *Options) (zapcore.Core, zapcore.Core, zapcore.WriteSyncer, error) {
-	encCfg := zapcore.EncoderConfig{
-		TimeKey:        "time",
-		LevelKey:       "level",
-		NameKey:        "scope",
-		CallerKey:      "caller",
-		MessageKey:     "msg",
-		StacktraceKey:  "stack",
-		LineEnding:     zapcore.DefaultLineEnding,
-		EncodeLevel:    zapcore.LowercaseLevelEncoder,
-		EncodeCaller:   zapcore.ShortCallerEncoder,
-		EncodeDuration: zapcore.StringDurationEncoder,
-		EncodeTime:     formatDate,
-	}
-
 	var enc zapcore.Encoder
-	if options.JSONEncoding {
+	if options.useStackdriverFormat {
+		encCfg := zapcore.EncoderConfig{
+			TimeKey:        "timestamp",
+			LevelKey:       "severity",
+			NameKey:        "logger",
```

Comment:
I don't see this key defined in the format above?

Reply:
Logger isn't in the LogEntry proto; that is correct.

However, it ensures that the Scope name is consistently placed in the jsonPayload.logger field. This matches the code for the direct-to-stackdriver bits (here: https://github.com/istio/pkg/pull/318/files#diff-468565f13023bc1ab3a3eb1dd6930ae69140cb051c3fedc7f2b9f9335319ae23R133). 

So, that bit is more about consistency. If you think we should use a different field name, let's do that.